src/interactive-auth.js

```diff
@@ -101,6 +107,7 @@ function InteractiveAuth(opts) {
     this._matrixClient = opts.matrixClient;
     this._data = opts.authData || {};
     this._requestCallback = opts.doRequest;
+    this._setBusyCallback = opts.setBusy;
```

Comment:
Maybe `busyChanged` for the name of the option key? Not sure how callbacks are typically named in JS SDK...